### DISCUSSION TITLE
Game causing a full PC lockup

### DISCUSSION THREAD

**Discussion by Failquail** [voxeltycoon/issues#637] [Troubleshooting]
**Describe the bug**
Haven't played for a while, but on trying the new steam edition the game locks up my PC within around 10minutes.

Lockup as in the monitors display a white screen, the sound repeats about half a second of audio repeatedly and then the PC reboots itself a few seconds later. No other game i own has done this before.

**Steps to reproduce**
Load a game, PC will consistently lockup within around 10 minutes, even if i don't touch the keyboard

**Build version**
0.85.1

**Platform**
Win 10 pro 20H2

**Reproducibility**
Every time, within about 10mins of playtime

**Logs/saves**
[Player.log](https://github.com/voxeltycoon/bug-tracker/files/6377853/Player.log)
(Not sure how usefull that'll be, as the system locks up completely)

Additional note:
This only happens on my desktop (R7 3700X, RX580 GPU)
My laptop works fine without issue (R7 2700U, onboard Vega10 GPU)

Also this wasn't a problem with the itch.io 0.84.1 version. (that still runs perfectly on this desktop PC)

**Comment by PiLdacz** [comment]
Hi, 
I have the same problem as you.
